countries/src/main/java/com/lambdaschool/countries/CountryController.java

```diff
@@ -0,0 +1,81 @@
+package com.lambdaschool.countries;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+
+@RestController
+@RequestMapping("/data")
+public class CountryController
+{
+    // localhost:7000/data/names/all
+    @RequestMapping(value = "/names/all")
+    public ResponseEntity<?> getAllCountries()
+    {
+        CountriesApplication.ourCountries.countryList.sort((c1, c2) -> c1.getName().compareToIgnoreCase(c2.getName()));
+        return new ResponseEntity<>(CountriesApplication.ourCountries.countryList, HttpStatus.OK);
+    }
+
+    // localhost:7000/data/names/start/s
+    @GetMapping(value = "/names/start/{letter}")
+    public ResponseEntity<?> getCountryLetter(@PathVariable char letter)
+    {
+        ArrayList<Country> rtnCountry = CountriesApplication.ourCountries
+                .findCountries(c -> c.getName().toUpperCase().charAt(0) == Character.toUpperCase(letter));
```

Comment:
Lambda expression looks awesome! Great work matching the first character to what was passed in